spec/system/distributions_by_county_system_spec.rb

```diff
@@ -0,0 +1,105 @@
+RSpec.feature "Distributions by County", type: :system do
+  let(:default_params) do
+    {organization_id: @organization.to_param}
+  end
+
+  let(:year) { Time.current.year }
+  let(:issued_at_last_year) { Time.current.utc.change(year: year - 1).to_datetime }
+  let(:issued_at_present) { Time.current.utc.to_datetime }
+  let(:item_1) { create(:item, value_in_cents: 1050) }
+
+  before do
+    sign_in(@user)
+    setup_overlapping_partners
+    @storage_location = create(:storage_location, organization: @organization)
+    setup_storage_location(@storage_location)
+  end
+
+  it "shows 'Unspecified 100%' if no served_areas" do
+    @distribution = create(:distribution, :with_items, item: item_1, organization: @user.organization)
+    visit distributions_by_county_report_path
```

Comment:
Can we move most of these from system to request specs? System specs are only really necessary if we're testing multiple pages (via clicks) or JavaScript. Simple visits should be doable with request specs, which don't fire up headless Chrome and hence are much faster.